app/Resources/views/admin/blog/show.html.twig

```diff
@@ -4,30 +4,13 @@
 
 {% block main %}
     <h1>{{ post.title }}</h1>
+    <p class="text-muted">{{ post.publishedAt|localizeddate('long', 'medium', null, 'UTC') }}</p>
```

Comment:
Wow! I totally missed this comment ... and I almost copied your idea here!  Yes, we could display that information ... but for the users I'd like to display people names instead of emails (so we might need to add a field in the user entity).